locale/en/submission.po

```diff
@@ -2043,6 +2043,28 @@ msgstr "You must assign at least one participant to this submission before initi
 msgid "submission.query.allowedEditTime"
 msgstr "You can update this discussion for {$allowedEditTimeNoticeLimit} minutes."
 
+msgid "submission.list.changeSubmissionLanguage.currentLanguage"
+msgstr "Current Submission Language: <strong>{$language}</strong>"
+
+msgid "submission.list.changeSubmissionLanguage.buttonLabel"
+msgstr "Change"
+
+msgid "submission.list.changeSubmissionLanguage.title"
+msgstr "Change Submission Language For"
+
+msgid "submission.list.changeSubmissionLanguage.languageDescription"
+msgstr "This is the primary submission language. Changing this will have effects on the submission and the metadata entered"
```

Comment:
This and a few others don't have periods at the end of sentences. I think it's fine when it's just a word or phrase, but this and the others below should probably have them.

Reply:
This should be so according to some design rules -- I have already asked Devika :-)